cmd/kubeadm/app/phases/upgrade/health.go

```diff
@@ -179,6 +203,7 @@ func createJob(client clientset.Interface, cfg *kubeadmapi.ClusterConfiguration)
 		return false, nil
 	})
 	if err != nil {
+		// Return the error as a warning for this preflight check
```

Comment:
It is still an error instead of a warning yet. right?

Reply:
missed that, updated.